src/ucp/rndv/rndv_ats.c

```diff
@@ -57,15 +57,7 @@ ucp_proto_rndv_ats_proto_progress(uct_pending_req_t *uct_req)
 {
     ucp_request_t *req = ucs_container_of(uct_req, ucp_request_t, send.uct);
 
-    if (!(req->flags & UCP_REQUEST_FLAG_PROTO_INITIALIZED)) {
-        if (req->send.rndv.rkey != NULL) {
-            ucp_proto_rndv_rkey_destroy(req);
-        }
-
-        ucp_datatype_iter_cleanup(&req->send.state.dt_iter, UCP_DT_MASK_ALL);
-        req->flags |= UCP_REQUEST_FLAG_PROTO_INITIALIZED;
-    }
-
+    req->flags |= UCP_REQUEST_FLAG_PROTO_INITIALIZED;
```

Comment:
maybe don't need this flag for rndv/ats protocol since nothing is really initialized here?